cri/ocicni/cni_manager.go

```diff
@@ -77,6 +77,9 @@ func (c *CniManager) SetUpPodNetwork(podNetwork *ocicni.PodNetwork) error {
 func (c *CniManager) TearDownPodNetwork(podNetwork *ocicni.PodNetwork) error {
 	err := c.plugin.TearDownPod(*podNetwork)
 	if err != nil {
+		if _, err = os.Stat(podNetwork.NetNS); err != nil {
```

Comment:
There is a more clear codestyle:

```
if err == nil {
      return nil
}

if _, err = os.Stat(podNetwork.NetNS); err != nil {
          return err
}
return fmt.Errorf("failed to destroy network for sandbox %q: %v", podNetwork.ID, err)
```
Decrease the nested layers is a quie important rule in clean codes

Reply:
agree

Reply:
Agree with that, THX.